raku-skeletons.el

```diff
@@ -0,0 +1,101 @@
+;;; raku-skeletons.el --- Skeletons for Raku file auto insertion.  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020  Tim Van den Langenbergh
+
+;; Author: Tim Van den Langenbergh <tmt_vdl@gmx.com>
+;; URL: https://github.com/raku/raku-mode
+;; Keywords: languages, convenience, files
+;; Version: 0.1
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Some skeletons for auto-insertion in Raku files.
+
+;;; Code:
+(require 'raku-mode)
+
+(defgroup raku-skeletons nil
+  "Skeletons for Raku files."
+  :prefix "raku-skeleton-"
+  :group 'raku)
+
+;; Need the full path for the #!. Simply setting it to `raku' may not be ideal.
+(defcustom full-raku-path "/usr/bin/raku"
```

Comment:
I am not sure what does it mean, but the string alone looks unsafe to me.
Am I right it is used to define a shebang below?
I think `#!/usr/bin/env raku` or `#!/usr/bin/env perl6` (tried in this order) would be more safe against various installations where the executable isn't put to /usr/bin.

Reply:
You're right, it's the default line we use for the shebang (it's a defcustom, so the user can change that in the settings). It can safely be set to "/usr/bin/env raku", and if the user has a need to they can change it to suit their needs. I'll push a commit in a jiffy.